src/components/UserInfo.js

```diff
@@ -1,20 +1,26 @@
 export default class UserInfo {
-  constructor({ nameSelector, jobSelector }) {
+  constructor({ nameSelector, jobSelector, avatarSelector }) {
     this._name = document.querySelector(nameSelector);
     this._job = document.querySelector(jobSelector);
+    this._avatar = document.querySelector(avatarSelector);
   }
 
   //returns an object containing information about the user
   getUserInfo() {
     return {
       title: this._name.textContent,
       description: this._job.textContent,
+      avatar: this._avatar.src,
     };
   }
 
   //takes new user data and adds it to the page
-  setUserInfo({ title, description }) {
-    this._name.textContent = title;
-    this._job.textContent = description;
+  setUserInfo({ name, about }) {
+    this._name.textContent = name;
+    this._job.textContent = about;
+  }
+
+  setUserAvatar(link) {
```

Comment:
It's great to have a separate setter for the avatar as it's controlled by the different form